release-notes/unreleased/IGAPP-1028-poi-markers-pixelated.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1028
+show_in_stores: true
+platforms:
+  - web
```

Comment:
Is this really web only? I think the used icons are changed on native as well, right?

Reply:
gonna adjust it